cfme/configure/configuration/region_settings.py

```diff
@@ -981,8 +981,7 @@ class ReplicationGlobalAddView(ReplicationView):
     username = Input(name='userid')
     password = Input(name='password')
     accept_button = Button('Accept')
-    action_dropdown = Dropdown(
-        "//*[@id='form_div']//table//button[contains(@class, 'dropdown-toggle')]")
+    action_dropdown = Dropdown("")
```

Comment:
Dropdown class required text but the current locator is not existed so passed as an empty string only.

Reply:
I'm very skeptical of this use of the widget without any title/text.  We may need an update to the widget class, but is there no title attribute or text for the component in the DOM?